libres/lib/include/ert/enkf/summary_obs.hpp

```diff
@@ -30,14 +30,20 @@
 #include <ert/enkf/meas_data.hpp>
 #include <ert/enkf/summary_config.hpp>
 #include <ert/enkf/summary.hpp>
-#include <ert/enkf/active_list.hpp>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 typedef struct summary_obs_struct summary_obs_type;
 
+extern "C++" {
+#include <ert/enkf/active_list.hpp>
```

Comment:
This should have C++ linkage if placed where it used to be?

Reply:
OK